xml/System.Runtime.InteropServices/SequenceMarshal.xml

```diff
@@ -176,9 +176,15 @@
         <typeparam name="T">To be added.</typeparam>
         <param name="reader">To be added.</param>
         <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Tries to read the given type out of the buffer if possible.
+Warning: this is dangerous to use with arbitrary structs - see remarks for full details.</summary>
+        <returns><see langword="true" /> if successful, <see langword="false" /> otherwise. <paramref name="value" /> will be default if failed (due to lack of space).</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+[!IMPORTANT]
+The read is a straight copy of bits. If a struct depends on specific state of it's members to behave correctly this can lead to exceptions, etc. If reading endian specific integers, use the explicit overloads such as <xref:System.Buffers.SequenceReaderExtensions.TryReadLittleEndian(System.Buffers.SequenceReader{System.Byte}@,System.Int32@)>.
```

Comment:
```suggestion
The read is a straight copy of bits. If a struct depends on the specific state of its members to behave correctly, this can lead to exceptions. If you're reading endian specific integers, use the explicit overloads such as <xref:System.Buffers.SequenceReaderExtensions.TryReadLittleEndian(System.Buffers.SequenceReader{System.Byte}@,System.Int32@)>.
```